Gemfile

```diff
@@ -12,7 +12,18 @@ end
 
 group :test do
   # ActiveSupport required to test compatibility with ActiveSupport Core Extensions.
-  gem 'activesupport', '~> 4.x', require: false
+  if RUBY_VERSION >= '2.2.2'
+    gem 'activesupport', '~> 5.x', require: false
```

Comment:
Hmm, this is a new bundler syntax to me. I haven't seen a `.x` version before. Is this the same as `~> 5`?